media-sound/flacon/flacon-5.5.1.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# Ignore rudimentary et, uz@Latn, zh_TW translation(s).
+PLOCALES="cs cs_CZ de el es es_MX fr gl hu it ja_JP lt ms_MY nb nl nl_BE pl pl_PL pt_BR pt_PT ro_RO ru sr sr@latin tr uk zh_CN"
+# Tests require lots of disk space
+CHECKREQS_DISK_BUILD=10G
+
+inherit check-reqs cmake-utils eutils gnome2-utils l10n virtualx xdg-utils
+
+DESCRIPTION="Extracts audio tracks from an audio CD image to separate tracks"
+HOMEPAGE="https://flacon.github.io/"
+SRC_URI="https://github.com/flacon/flacon/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test +ape +tta +mp3gain +vorbisgain"
+
+RDEPEND="
+	app-i18n/uchardet
+	dev-qt/qtcore:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtwidgets:5
+"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+	dev-qt/linguist-tools:5
+	dev-qt/qtconcurrent:5
+	ape? ( media-sound/mac )
+	tta? ( media-sound/ttaenc )
+	mp3gain? ( media-sound/mp3gain )
+	vorbisgain? ( media-sound/vorbisgain )
+	test? (
+		dev-qt/qttest:5
+		media-libs/flac
+		media-sound/mac
+		media-sound/shntool
+		media-sound/ttaenc
+		media-sound/wavpack
+	)
+"
```

Comment:
Please sort the *DEPEND entries also in latin alphabetical order (starting with the first letter, so for "ape?" with a, virtual/pkgconfig v and so on).